solidity/test/isms/IsmTestUtils.sol

```diff
@@ -17,7 +17,7 @@ library MessageUtils {
         uint32 _destinationDomain,
         bytes32 _recipient,
         bytes memory _messageBody
-    ) private pure returns (bytes memory) {
+    ) public pure returns (bytes memory) {
```

Comment:
why aren't we just using the `Message` library for this

Reply:
need memory instead of calldata

Reply:
https://github.com/hyperlane-xyz/hyperlane-monorepo/blob/16146f0f03d0b0be67583cf16b22a3c50abdd977/solidity/contracts/libs/Message.sol#L33